fix_framework.py

```
@@ -0,0 +1,57 @@
+    'src/framework_persistence.rs',
+    'src/framework_rerank.rs',
+    'src/framework_validation.rs',
+    'src/framework_ttl.rs'
+]
+
+for p in files:
+    if os.path.exists(p):
+        fix_file(p)
+
+# Fix CLI commands - use HVec10240 for now to maintain behavior
+with open('src/cli/commands/mod.rs', 'r') as f:
```

Ruff (0.15.12)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)